docs/src/main/paradox/migration-guide.md

```diff
@@ -14,6 +14,18 @@ DROP CONSTRAINT durable_state_pkey,
 ADD PRIMARY KEY(persistence_id);
 ```
 
+Same goes for the (optional) index `` used for slice base queries. To remove the column from the index, use:
+
+Postgres / Yugabyte:
+: ```sql
+DROP INDEX IF EXISTS durable_state_slice_idx;
+CREATE INDEX IF NOT EXISTS durable_state_slice_idx ON durable_state(slice, entity_type, db_timestamp);
+```
```

Comment:
Maybe we should clarify that this migration is optional. If someone is already using it I don't think it would hurt much to just continue with the existing index. Maybe same with the primary key. As mentioned below, there can be a huge performance impact on the db to recreate the index if the data volume is large.

Reply:
Also, see that this section is currently under the following headline:

> ### Durable State table schema change (optional)